mdoc/src/main/scala/mdoc/internal/cli/Settings.scala

```diff
@@ -161,7 +161,11 @@ case class Settings(
   def isFileWatching: Boolean = watch && !check
 
   def toInputFile(infile: AbsolutePath): Option[InputFile] = {
-    val relpath = infile.toRelative(in)
+    val relpath = if (infile == in) {
+      RelativePath(in.toFile.getName)
```

Comment:
Let's avoid using the `java.io.File` API, the `java.nio.file` API can be more verbose but it works better in my experience
```suggestion
      RelativePath(in.toNIO.getFileName.toString)
```

Reply:
Sounds good 👍